.cirrus.yml

```diff
@@ -1,3 +1,35 @@
+task:
```

Comment:
you should name these tests:
`lint_task` and `test_task`

Reply:
but that will be reduced same we have right according to docs. Still want?

Reply:
ah, you're right, nvm